### PR TITLE
Droth 1599 tn its warning sign

### PR DESCRIPTION
**Pull request by MAC-CGI**
Droth 1599 tn its warning sign

### CODE REVIEW

**Comment by joanacoelho** [comment]
`kauniainen_obstacles.sql`

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala [hunk 1]
```diff
@@ -238,6 +242,28 @@ class TrafficSignService(val roadLinkService: RoadLinkService, val userProvider:
     }
   }
 
+  override def getChanged(sinceDate: DateTime, untilDate: DateTime): Seq[ChangedPointAsset] = { throw new UnsupportedOperationException("Not Supported Method, Try to used") }
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala]: This method should be replace by 
**getChanged( sinceDate: DateTime, untilDate: DateTime, trafficSignTypes: Set[Int] = Set())** on base and could be remove the Unsupported exception

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ChangeApi.scala [hunk 2]
```diff
@@ -138,6 +143,67 @@ class ChangeApi extends ScalatraServlet with JacksonJsonSupport with Authenticat
...
+    "modifiedAt" -> point.modifiedAt.map(DateTimePropertyFormat.print(_)),
+    "createdAt" -> point.createdAt.map(DateTimePropertyFormat.print(_)),
+    "modifiedBy" -> point.modifiedBy,
+    "changeType" -> extractChangeType(since, point.expired, point.createdAt),
+    "sideCode" -> SideCode.BothDirections.value
+   )
+  }
+
+  def pointAssetTrafficSignProperties(pointAsset: PersistedPointAsset, since: DateTime): Map[String, Any] = {
+    val point = pointAsset.asInstanceOf[PersistedTrafficSign]
```
[the remark this answers is not in the document]
Comment (marquesfm) [reply digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ChangeApi.scala]: This code was already simplified on DROTH-1598.

**Comment by oskarmalmstrom** [comment]
Following branches were merged into one branch TNITS_deploy and deployed. Closing this pull request without merging.  
- DROTH-486
- DROTH-1575
- DROTH-507
- DROTH-1599
- DROTH-1598
- DROTH-1612

**Pull request closed without merging**